infrastructure/docker/builder/Makefile

```diff
@@ -21,10 +21,10 @@ IMAGENAME?=$(IMAGE_REGISTRY)/kuberpult-builder:${VERSION}
 build:
 	docker build -t $(IMAGENAME) .
 
-build-pr: build publish
```

Comment:
i guess this is for easier testing of the PR?
or do we not need the `login` target anymore?

Reply:
I do the login in the ci/cd pipeline... and it became tricker, because we now need to login in two places